pod/perlop.pod

```diff
@@ -1488,14 +1492,12 @@ expressions.  It's equivalent to C<&&> except for the very low
 precedence.  This means that it short-circuits: the right
 expression is evaluated only if the left expression is true.
 
-=head2 Logical or and Exclusive Or
-X<operator, logical, or> X<operator, logical, xor>
-X<operator, logical, exclusive or>
-X<or> X<xor>
+=head2 Logical Or
+X<operator, logical, or> X<or>
 
-Binary C<"or"> returns the logical disjunction of the two surrounding
-expressions.  It's equivalent to C<||> except for the very low precedence.
-This makes it useful for control flow:
+Logical C<"or"> returns the logical inclusive disjunction of the two
```

Comment:
I do not think "binary" here refers to the operation, but to the syntax of the operator: it takes two arguments. Note that the versions of these operators marked as "bitwise" are also introduced with this term.